spec/requests/categories/update_spec.rb

```diff
@@ -0,0 +1,38 @@
+RSpec.describe 'Category update', type: :request do
+  let(:category) do
+    create(:category, name: 'Some category')
+  end
+
+  context 'with valid params' do
+    let(:params) do
+      { category: { name: "Another category" } }
+    end
+
+    it 'updates category ' do
+      patch "/categories/#{category.id}", params: params
+
+      expect(response).to redirect_to(categories_path)
+      follow_redirect!
+
+      category.reload
+
+      expect(category.name).to be_eq('Another category')
```

Comment:
Wait. Why don't use same logic as in create spec?
```suggestion
      expect(category).to have_attributes(params[:category])
```